source/includes/steps-enable-authentication-in-shardcluster-nodowntime-transition-mongos.yaml

```diff
@@ -50,17 +51,20 @@ pre: |
   method against the ``admin`` database to safely shut down the
   :program:`mongos`.
   
-  Restart one :program:`mongos` at a time to ensure that clients
-  can connect to the sharded cluster. If your cluster has only one
-  :program:`mongos`, this step results in downtime while the :program:`mongos`
-  is offline.
+  Restart one :program:`mongos` at a time to ensure that clients can connect
+  to the sharded cluster. If your cluster has only one :program:`mongos`, this
+  step results in downtime while the :program:`mongos` is offline.
+  
+  Connect to the :program:`mongos` and use the :method:`db.shutdownServer()`
+  method against the ``admin`` database to safely shut down the
+  :program:`mongos`.
   
   .. code-block:: javascript
   
      db.getSiblingDB("admin").shutdownServer()
   
-  You can then start the :program:`mongos` with the new configuration
-  file, specifying the path to the config file using :option:`--config`.
+  Restart :program:`mongos` with the new configuration file, specifying the
+  path to the config file using :option:`--config`.
```

Comment:
My logic was to use example blocks for when I want to set up a scenario or context for a code example, as opposed to the code block here which needs no context (IMHO).